### DISCUSSION TITLE
Howto re-set ui.range min/max boundaries and apropriate value programatically?

### DISCUSSION THREAD

**Discussion by bojh** [zauberzeug/nicegui#3167] [Q&A]
### Question

I want to programmatically re-set the ui.range min/max boundaries and min/max values. 
```python
from nicegui import app, ui

my_range = ui.range(min=0, max=100, value={'min':5, 'max':95})
my_value = ui.label().bind_text_from(my_range, 'value', 
    backward=lambda v: f'v_min: {v["min"]}, v_max: {v["max"]}')

def set_bnd_val(b_min, b_max, v_min, v_max):
        my_range.props((f'min={b_min} max={b_max}'))
        my_range.value = {'min':v_min, 'max':v_max}

my_button_set1 = ui.button("value In: [40- 60]", on_click=lambda: set_bnd_val(40, 60, 45,55))
my_button_set2 = ui.button("value In: [ 0-100]", on_click=lambda: set_bnd_val( 0,100,  5,95))

ui.run()
```
... pressing buttons seems working well. The range slider does update and the new proportion is OK. 
The troubles come up touching the value handles: The reported values does create 'None' values, depending which handle is moved and the slider is no longer correct.

The question is what is wrong or missing to do it in the right way?

**Comment by falkoschindler** [comment]
Hi @bojh, this is very strange! Setting the boundaries seems to break the range element:
```py
r = ui.range(min=0, max=100, value={'min': 30, 'max': 70})
ui.button('Change limits', on_click=lambda: r.props('min=10 max=90'))
```

Is this a Quasar bug? Can we reproduce it with a plain HTML/Quasar demo?

**Comment by bojh** [comment]
Had a look-up to the quasar component. I'll experiment with it's 'inner-mn' and 'inner-max' options.
see: [Quasar inner-min inner-max](https://quasar.dev/vue-components/range#with-inner-min-max)

**Comment by falkoschindler** [comment]
I created issue #3191 to further track this bug.
